src/pages/actor_detail.js

```
@@ -0,0 +1,92 @@
+
+import { getActor } from "../services/actorServices";
+
+import DetailPageLoading from "../components/detail_page-loading";
+import ActorFilms from "../components/actor_films";
+
+import { FaRegEye } from "react-icons/fa";
+import { GiBodyHeight } from "react-icons/gi";
+import { LiaBirthdayCakeSolid } from "react-icons/lia";
+import { IoIosReturnLeft } from "react-icons/io";
+import Avatar from "../components/avatar";
+import Error from "../components/error";
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)